src/cli.ts

```diff
@@ -30,6 +30,10 @@ program
     })
   )
   .option('-s, --silent', 'silent all logs')
+  .option('--cairo', 'Run a Cairo program')
```

Comment:
I'm still wondering why this is required. I think that we should have one `Program` type and that at the VM level, there is no Cairo/CairoZero but CASM. If for now we can't (for some reasons to be better explained) have this, at lease the parser of the input should be able to figure this out